### ISSUE TITLE
Parse error on shorthand arguments forwarding

### ISSUE THREAD

**Issue by doits** [presidentbeef/brakeman#1449]
### Background

Brakeman version: 4.7.2
Rails version: 6.0.2.1
Ruby version: 2.7.0p0

Link to Rails application code: -

#### Parse Error

It fails to parse ruby's 2.7 new arguments forwarding shortand `...` (see https://blog.saeloun.com/2019/12/04/ruby-2-7-adds-new-operator-for-arguments-forwarding.html for some background information).

Minimal example that does not parse:

```ruby
def some_method(...)
  some_other_method(...)
end

def some_other_method(name)
  pp name
end
```

```
Error: path/to/file.rb:1 :: parse error on value "..." (tDOT3)
```

**Comment by presidentbeef** [comment]
Brakeman relies on `ruby_parser` for parsing Ruby. Naturally, support will lag a little bit for brand-new syntax.

**Comment by presidentbeef** [comment]
This form is addressed in `brakeman` 5.0.1 gem.

(The newer form where you can have regular arguments and `...` is not supported yet.
